public/app/features/alerting/unified/components/rule-editor/AlertRuleForm.tsx

```diff
@@ -244,7 +244,6 @@ export const AlertRuleForm = ({ existing, prefill, id }: Props) => {
                 <>
                   {type === RuleFormType.grafana ? (
                     <GrafanaEvaluationBehavior
-                      initialFolder={defaultValues.folder}
```

Comment:
folder is already in the form's `defaultValues` and fetch from the form context is `AlertGroup` so I'm struggling to figure out why it was being passed as a separate prop 🤔